src/enums/sortingEnum.ts

```diff
@@ -0,0 +1,4 @@
+export enum SortType {
+  Alphabetically,
+  ByLength,
+}
```

Comment:
The enum `SortType` is well-named and follows the checklist item 4, as it's a noun representing a set of options. However, please ensure that the enum values are used appropriately in the rest of the application to avoid direct interaction with the DOM, as per checklist item 1.